stringlab.js

```diff
@@ -0,0 +1,75 @@
+// console.log the first character of a string.
```

Comment:
Overall I can see that you understand string methods and properties. Please note comments below, go over some of problems again. Also try to do bonus questions next time as they are good practice to improve technical mastery. Keep practicing and keep up the good work!